mantle/cmd/kola/kola.go

```diff
@@ -173,6 +189,51 @@ func runRun(cmd *cobra.Command, args []string) error {
 		patterns = args
 	}
 
+	return kolaRunPatterns(patterns, runRerunFlag)
+}
+
+func runRerun(cmd *cobra.Command, args []string) error {
+	var patterns []string
+	type jsonTest struct {
+		Name   string                `json:"name"`
+		Result testresult.TestResult `json:"result"`
+		Output string                `json:"output"`
+	}
+	type jsonSuiteResult struct {
```

Comment:
Shouldn't we reuse the existing types for these?

Reply:
I wasn't sure if we wanted to change the visibility on the existing types to public for just this one use. I'm open to changing this if needed.

Reply:
Alternatively, we could add a method like `NewJSONReporterFromFile()` that deserializes a path, and something like a `GetFailed()` method on the `jsonReporter` object?